tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "allowJs": true,
     "jsx": "react-jsx",
     "noEmit": true,
-    "strict": false
-  }
+    "incremental": true
```

Comment:
`strict` is enabled by default in the tsconfig we extend.
`incremental` seemed like a no-brainer to enable. Basically a type checking cache. We should probably do this for the sku tsconfig as well.